app/Resources/translations/default/ModulesAutoupgradeAdmin.xlf

```diff
@@ -1355,4 +1487,1037 @@ File: modules/autoupgrade/classes/TaskRunner/AbstractTask.php:114</note>
+      </trans-unit>
+      <trans-unit id="48995d15304fe79fc0240bf6544f2395">
+        <source>Are you sure you want to delete this backup?</source>
+        <target>Are you sure you want to delete this backup?</target>
+        <note>Context:
+          File: modules/autoupgrade/classes/UpgradePage.php:324</note>
+      </trans-unit>
+      <trans-unit id="f03a04d1f440e359840035679b49b212">
+        <source>An update is currently in progress... Click "OK" to abort.</source>
+        <target>An update is currently in progress... Click "OK" to abort.</target>
+        <note>Context:
+          File: modules/autoupgrade/classes/UpgradePage.php:326</note>
+      </trans-unit>
+      <trans-unit id="98a399b35ec9e03ec985254acfe5e3a0">
+        <source>Upgrading PrestaShop</source>
+        <target>Upgrading PrestaShop</target>
+        <note>Context:
+          File: modules/autoupgrade/classes/UpgradePage.php:327</note>
+      </trans-unit>
+      <trans-unit id="e02d726a7e733ffb7737e53c89be5e4f">
+        <source>Upgrade complete</source>
+        <target>Upgrade complete</target>
+        <note>Context:
+          File: modules/autoupgrade/classes/UpgradePage.php:328</note>
+      </trans-unit>
+      <trans-unit id="f67d595b77c8c85da5434ddf41b840d4">
+        <source>Upgrade complete, but warning notifications has been found.</source>
+        <target>Upgrade complete, but warning notifications has been found.</target>
+        <note>Context:
+          File: modules/autoupgrade/classes/UpgradePage.php:329</note>
+      </trans-unit>
+      <trans-unit id="a9cb4ff8ba5698ccdc9672435be31d24">
+        <source>Cookies have changed, you will need to log in again once you refreshed the page</source>
+        <target>Cookies have changed, you will need to log in again once you refreshed the page</target>
+        <note>Context:
+          File: modules/autoupgrade/classes/UpgradePage.php:331</note>
+      </trans-unit>
+      <trans-unit id="546c6a7b5ca345af3880bfc74a339606">
+        <source>Javascript and CSS files have changed, please clear your browser cache with CTRL-F5</source>
+        <target>Javascript and CSS files have changed, please clear your browser cache with CTRL-F5</target>
+        <note>Context:
+          File: modules/autoupgrade/classes/UpgradePage.php:332</note>
+      </trans-unit>
+      <trans-unit id="19ea4327ee1bad0ed94b42a77255f3c6">
+        <source>Please check that your front-office theme is functional (try to create an account, place an order...)</source>
+        <target>Please check that your front-office theme is functional (try to create an account, place an order...)</target>
+        <note>Context:
+          File: modules/autoupgrade/classes/UpgradePage.php:333</note>
+      </trans-unit>
+      <trans-unit id="c12803674a35ccc28e5da00a50a84591">
+        <source><![CDATA[Product images do not appear in the front-office? Try regenerating the thumbnails in Preferences > Images]]></source>
```

Comment:
The wording looks the same to me. Am I missing something? The `<![CDATA[]]>` around is syntactic sugar to prevent the literal `>` inside the wording from breaking the XML file. You can disregard that, it won't appear in Crowdin.